### ISSUE TITLE
findrepe is always using unix path separator is 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#7]
```
findrepe is always using unix path separator, so it is not excluding paths
with ":" in windows like "c:\windows"

it should use path separator defined in File.pathSeparator

```

Original issue reported on code.google.com by `flik...@gmail.com` on 27 Jun 2009 at 10:24

**Comment by GoogleCodeExporter** [comment]
```
fixed, version 0.6.0
```

Original comment by `flik...@gmail.com` on 4 Jul 2009 at 12:32
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 10 Jan 2010 at 11:17
- Changed state: **Verified**
